azure-pipelines.yml

```diff
@@ -174,7 +177,7 @@ stages:
       displayName: Build Uno Apps
       inputs:
         solution: $(build.sourcesdirectory)/NuGetPackageExplorer-uno.slnf
-        msbuildArguments: /r /p:Configuration=Release /p:UNO_BUILD_ONLY=true /bl:$(Build.ArtifactStagingDirectory)\Logs\$(ReleaseChannel).binlog
+        msbuildArguments: /r /p:Configuration=Release /p:UNO_BUILD_ONLY=true /p:NpeAiInstrumentationKey=$(NPE_AI_INSTRUMENTATIONKEY) /bl:$(Build.ArtifactStagingDirectory)\Logs\$(ReleaseChannel).binlog
```

Comment:
There's already a variable for this in the pipeline, though it's not in its own variable group yet:

https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/main/azure-pipelines.yml#L116-L121

Reply:
if we can do it in a script instead of a build variable, it keeps it out of the binlog too.

Reply:
I don't think we'll be able to put it in a script, since it has to be an msbuild variable to become an wasm environment variable.

Reply:
What if we put a placeholder `REPLACE_ME_WITH_THE_KEY` in the msbuild variable and replace it in the generated output?

Reply:
The output is compressed by the tooling, we'd need to re-generate the compressed file as well, or delete it so the uncompressed file takes over. Still, the key is easily readable from the live app, so changing this won't make it more secret.

Reply:
Gotcha, so be it.